Automattic/wp-calypso
# Domains: Update support URLs

#### Proposed Changes
Updates the domains' old links in the [support URLs file](client/lib/url/support.js).

#### Testing Instructions
Open the links and make sure that they are pointing to the right section/in-page anchors. 

Related to #64285.
